src/pages/mobile/Redeem.vue

```diff
@@ -0,0 +1,192 @@
+          v-validate="'required|address'"
+          autofocus
+          :error="errors.has('accountTo')"
+          :footer="errors.first('accountTo') || sendable ? $t('transfer.redeem-balance.error') : ''"
+          name="accountTo"
+          :header="$t('transfer.send.to.recipient')"
+        />
+      </form>
+    </template>
+
+    <DetailsAmountCurrency
+      :name="$t('transfer.send.amount.balance')"
+      :amount="balance"
+    />
+
+    <AeButton
+      :disabled="errors.any() || sendable"
+      :form="_uid"
+      fill="secondary"
+    >
+      {{ $t('confirm') }}
+    </AeButton>
+
+    <ListItemAccount
+      v-for="account in accounts"
+      :key="account.address"
+      border-dark
+      v-bind="account"
+      @click="accountTo = account.address"
+    >
+      <LeftMore slot="right" />
+    </ListItemAccount>
+  </MobilePage>
+</template>
+
+<script>
+import { pick } from 'lodash-es';
+import BigNumber from 'bignumber.js';
+import { Universal, MemoryAccount } from '@aeternity/aepp-sdk/es';
+import {
+  generateKeyPairFromSecret,
+  hexStringToByte,
+  aeEncodeKey,
+} from '@aeternity/aepp-sdk/es/utils/crypto';
+import AeLoader from '../../components/AeLoader.vue';
+import MobilePage from '../../components/mobile/Page.vue';
+import Guide from '../../components/Guide.vue';
+import AccountInline from '../../components/AccountInline.vue';
+import AeInputAccount from '../../components/AeInputAccount.vue';
+import DetailsAmountCurrency from '../../components/mobile/DetailsAmountCurrency.vue';
+import AeButton from '../../components/AeButton.vue';
+import ListItemAccount from '../../components/ListItemAccount.vue';
+import { LeftMore } from '../../components/icons';
+import { MIN_SPEND_TX_FEE, MAGNITUDE } from '../../lib/constants';
+
+export default {
+  components: {
+    AeLoader,
+    MobilePage,
+    Guide,
+    AccountInline,
+    AeInputAccount,
+    DetailsAmountCurrency,
+    AeButton,
+    ListItemAccount,
+    LeftMore,
+  },
+  data() {
+    return {
+      accountTo: '',
+      balance: 0,
+      busy: true,
+      sendable: true,
+      client: {},
+    };
+  },
+  subscriptions() {
+    return pick(this.$store.state.observables, ['accounts']);
+  },
+  async mounted() {
+    let privateKey;
+    try {
+      privateKey = await this.$store.dispatch('modals/open', {
+        name: 'readQrCode',
+        title: this.$t('transfer.redeem-balance.scan'),
+      });
+      const keypair = generateKeyPairFromSecret(hexStringToByte(privateKey));
+      this.address = aeEncodeKey(keypair.publicKey);
+      this.client = await Universal({
+        url: this.$store.state.sdkUrl,
+        compilerUrl: 'https://compiler.aepps.com',
+        accounts: [MemoryAccount({
+          keypair: {
+            secretKey: privateKey,
+            publicKey: this.address,
+          },
+        })],
+        address: this.address,
+      });
+    } catch (e) {
+      this.$router.push({ name: 'transfer' });
+      this.busy = false;
```

Comment:
It looks not necessary because after navigation this component will be destroyed.